### PR TITLE
Call TestUtilities.InitUtilities appropriately

### PR DESCRIPTION
**Pull request by basoundr**
Calling TestUtilities.InitUtilities from within the library, instead of a csx file, will result in the path of the file where the library was initially built to be passed to InitUtilities which is not the expected behavior. Fixing it in all occurence

Also removing some unnecessary and unused helpers

Tagging @KevinH-MS @rchande @TyOverby for review

### CODE REVIEW

File: src/Test/Perf/util/TestUtilities.cs [hunk 1]
```diff
@@ -20,6 +20,15 @@ public class TestUtilities
...
+        /// <summary>
+        /// This method should be called *ONLY* by the csx scripts.
+        /// This method *MUST NOT* be called from within the library itself. If this method is called within the library instead of csx scripts
+        /// then <param name="sourceFilePath"/> will be set to the path of the file when the library is actual built.
+        /// For Eg: If SomeLibarayFile.cs calls this method and if the library is built is some build machine where SomeLibarayFile.cs is saved at
+        /// Y:/Project/SomeLibarayFile.cs then when the library is used in any machine where there is no Y: drive then we will see errors saying
+        /// invalid directory path. Also note that <param name="sourceFilePath"/> will be set to "Y:/Project/SomeLibarayFile.cs" and not set to
+        /// the path of the file from where the call to SomeLibarayFile.cs which in turn called <see cref="InitUtilities(string)"/>
+        /// </summary>
         public static void InitUtilities([CallerFilePath] string sourceFilePath = "")
```
Comment (rchande) [inline src/Test/Perf/util/TestUtilities.cs]: `InitUtilitiesFromCsx`?

File: src/Test/Perf/util/TraceManager.cs [hunk 2]
```diff
@@ -63,24 +59,24 @@ public void Initialize()
 
         public void Setup()
         {
-            ShellOutVital(_cpcPath, "/Setup /DisableArchive", _verbose, _logger);
+            ShellOutVital(_cpcPath, "/Setup /DisableArchive", _verbose, _logger, workingDirectory: "");
```
Comment (rchande) [inline src/Test/Perf/util/TraceManager.cs]: Why?

**Comment by rchande** [comment]
:+1:

File: src/Test/Perf/util/TraceManager.cs (hunk 2 above)
Comment (basoundr) [inline src/Test/Perf/util/TraceManager.cs]: If we don't specify the value explicitly then we will try to run the command inside `MyWorkingDirectory()` directory. This scenario holds true only when called from csx files. 

In short, both calling `InitUtilities()` from within library and not setting the value of _workingDirectory are wrong. The correct approach is to always provide the directory to run the command if it is run outside of the context of csx.

File: src/Test/Perf/util/TestUtilities.cs (hunk 1 above)
Comment (basoundr) [inline src/Test/Perf/util/TestUtilities.cs]: sure

**Pull request merged**